client/my-sites/upgrades/checkout-thank-you/style.scss

```diff
@@ -145,31 +145,76 @@
 
 // Footer Subcomponent styles
 .checkout-thank-you__footer {
-	color: $gray;
+	padding: 32px;
+	text-align: left;
+
+	@include breakpoint( ">660px" ) {
+		padding-bottom: 48px;
+		padding-left: 24px;
+		padding-right: 24px;
+		padding-top: 48px;
```

Comment:
I agree with @scruffian that `padding: 48px 24px;` is a bit nicer :) If anything for consistency with the rest of our styles.